<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="txt_choix_pokemon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Choix du pokémon :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboBox_choix_pokemon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <item>
     <property name="text">
      <string>pokémon 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>pokémon 2</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="fond_ciel_bleu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>821</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/prairie_et_ciel.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="perso_qui_court">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>-30</y>
      <width>201</width>
      <height>371</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/perso_attaque.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image_pokemon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/images_pokemon/pokemons_finaux/face/Bulbasaur.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>fond_ciel_bleu</zorder>
   <zorder>txt_choix_pokemon</zorder>
   <zorder>ComboBox_choix_pokemon</zorder>
   <zorder>ok</zorder>
   <zorder>perso_qui_court</zorder>
   <zorder>image_pokemon</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
